libraries/botbuilder-dialogs/src/choices/channel.ts

```diff
@@ -11,20 +11,21 @@ import { TurnContext } from 'botbuilder-core';
  * @private
  */
 export const channels: any = {
+    console: 'console',
```

Comment:
We don't have an official console adapter, do we still need this? (We just have a sample...)

If someone were developing a bot for the console, I think they would know that cards, buttons, etc are not supported.